crates/starknet-devnet-server/src/api/serde_helpers.rs

```diff
@@ -1,18 +1,36 @@
 /// A module that deserializes `[]` optionally
 pub mod empty_params {
-    use serde::{Deserialize, Deserializer};
+    use serde::{de::Error as DeError, Deserialize, Deserializer};
+    use serde_json::Value;
```

Comment:
It's a shame you deleted the content of the PR template, it contains a reminder to format the code using `./scripts/format.sh`. I believe it has a different opinion on the order of imports here.

Reply:
I have now followed the template and added tests